geomstats/backend/tensorflow.py

```diff
@@ -185,3 +185,18 @@ def expand_dims(x, axis=None):
 
 def clip(x, min_value, max_value):
     return tf.clip_by_value(x, min_value, max_value)
+
+
+def diag(x):
+    return tf.linalg.diag(x)
```

Comment:
in tensorflow_linalg.py

Reply:
this is questionable, as it's just making a matrix with a specified vector on a diagonal

actually, it's inconsistent among backends:

- `np.diag`
- `torch.diag`
- `tf.linalg.diag`

by moving `tf.linalg.diag` to tensorflow_linalg.py the same function will be in the different files, which can be confusing

which consistency would we like to preserve more?

- same function, same backend file
- `backend.function` > backend.py, `backend.linalg.function` > backend.linalg.py

Reply:
You're right, let's keep it in `tensorflow.py` then! When there is an inconsistency among backends, we chose to follow numpy's. Thanks for catching this!